packages/fxa-settings/src/lib/channels/firefox.ts

```diff
@@ -337,7 +337,8 @@ export class Firefox extends EventTarget {
   }
 
   fxaCanLinkAccount(options: FxACanLinkAccount) {
-    this.send(FirefoxCommand.Login, options);
+    console.log('sending', FirefoxCommand.CanLinkAccount);
+    this.send(FirefoxCommand.CanLinkAccount, options);
```

Comment:
I'll remove this console log, but one thing I updated here was that we were sending `FirefoxCommand.Login` instead of `FirefoxCommand.CanLinkAccount`.

With this update, I'm uncertain if we want to send this event on signin and signup and I can remove it on this PR. Users that try to login to another account via Sync when they've already got a sync account set up will see the browser "Are you sure you want to Sync? Accounts will merge..." alert message after they click "submit" on the home screen, and then they'll see it once more once they've gotten to `/signin` due to us sending the event.

I think I assumed when working on Signup Sync that this webchannel message was needed to connect to Sync, but it _seems_ like we use it so the browser can show the user this prompt. We already have some notes to move this to the index screen once it's converted to React. I think I'll have time tomorrow to remove this and get tests done if that sounds preferable. @vbudhram

Reply:
@LZoog I'm not really sure on the dialog popping up twice? Leaning towards the current behavior where it would only be shown once.